실패율/main.py

```diff
@@ -0,0 +1,17 @@
+# 프로그래머스 실패율 문제 풀이
+# https://programmers.co.kr/learn/courses/30/lessons/42889
+
+def solution(N: int, stages: list):
+    # 실패한 사람 수를 리스트에 각 스테이지 마다 담음
+    fail_counts = [0 for i in range(0, N + 1)]
+    for stage in stages:
+        fail_counts[stage - 1] += 1
+    # 실패율을 계산하여 스테이지는 키, 실패율은 값에 담음
+    fail_rates = {}
+    for i in range(len(fail_counts) - 1):
+        if total := sum(fail_counts[i:]):
+            fail_rates[i] = fail_counts[i] / total
+        else:
+            fail_rates[i] = 0
```

Comment:
딕셔너리 컴프리헨션,,, 으로도 할 수 있다
```suggestion
    fail_rates = { i : fail_counts[i] / total 
              if (total := sum(fail_counts[i:])) else 0
              for i in range(len(fail_counts) - 1) 
             }
```

Reply:
제발 그만 해..... 이러다 다 죽어~~~

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 파이썬은 코드 줄이는 맛이지